docs/api/sdk/README.md

```diff
@@ -400,35 +401,45 @@ Use the `exception` type for errors and crashes.
 
 `exception` objects
 
-Each exception object contains further fields.
+Each exception object contains the thread information of the crashed thread and its stackframes along with the metadata required for symbolication.
 
-| Field     | Type   | Optional | Comment                     |
-| --------- | ------ | -------- | --------------------------- |
-| `type`    | string | No       | Type of the exception       |
-| `message` | string | No       | Error message text          |
-| `frames`  | array  | Yes      | Array of stackframe objects |
+| Field                      | Type    | Optional | Comment                        |
+| -------------------------- | ------- | -------- | ------------------------------ |
+| `type`                     | string  | Yes      | Type of the exception          |
+| `message`                  | string  | Yes      | Error message text             |
+| `frames`                   | array   | Yes      | Array of stackframe objects    |
+| `signal`                   | string  | Yes      | BSD termination signal         |
+| `thread_name`              | string  | Yes      | Name of thread                 |
+| `thread_sequence`          | Int     | Yes      | Thread number                  |
+| `operating_system_build`   | string  | Yes      | Operating system build number  |
```

Comment:
Nit: what do you think about naming this `os_build_number`?

Reply:
sounds good. will update.